cmd/cue-gen/test/generated/kubernetes/customresourcedefinitions.gen.yaml

```diff
@@ -19,40 +19,119 @@ spec:
     shortNames:
     - b
     singular: bar
-  preserveUnknownFields: false
   scope: Namespaced
-  subresources:
-    status: {}
-  validation:
-    openAPIV3Schema:
-      properties:
-        spec:
-          properties:
-            fieldA:
-              type: integer
-            fieldB:
-              format: string
-              type: string
-          type: object
-        status:
-          description: Status is the test status field.
-          properties:
-            analysis:
-              description: Analysis message.
-              format: string
-              type: string
-            condition:
-              description: Current state.
-              format: string
-              type: string
-          type: object
-      type: object
   versions:
   - name: v1alpha3
+    schema:
+      openAPIV3Schema:
+        properties:
+          spec:
+            properties:
+              fieldA:
+                type: integer
+              fieldB:
+                format: string
+                type: string
+            type: object
+          status:
+            description: Status is the test status field.
+            properties:
+              analysis:
+                description: Analysis message.
+                format: string
+                type: string
+              condition:
+                description: Current state.
+                format: string
+                type: string
+            type: object
+        type: object
     served: true
     storage: true
+    subresources:
+      status: {}
   - name: v1beta1
+    schema:
```

Comment:
oh man... are our CRDs going to be 2x as large now since the schema is duplicated?

Reply:
Not a real issue - just curious

Reply:
Yes, that is how the v1 CRD structures. Validation now lives under each version.